README.md

```diff
@@ -1,32 +1,89 @@
-# Node.js Client for Google Cloud Memorystore for Redis API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-node#versioning))
-
-[Google Cloud Memorystore for Redis API][Product Documentation]:
-The Google Cloud Memorystore for Redis API is used for creating and managing
-Redis instances on the Google Cloud Platform.
-- [Client Library Documentation][]
-- [Product Documentation][]
-
-## Quick Start
-In order to use this library, you first need to go through the following
-steps:
-
-1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
-2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
-3. [Enable the Google Cloud Memorystore for Redis API.](https://console.cloud.google.com/apis/api/redis)
-4. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-node/#/docs/google-cloud/master/guides/authentication)
-
-### Installation
-```
-$ npm install --save redis
-```
-
-### Next Steps
-- Read the [Client Library Documentation][] for Google Cloud Memorystore for Redis API
-  to see other available methods on the client.
-- Read the [Google Cloud Memorystore for Redis API Product documentation][Product Documentation]
-  to learn more about the product and see How-to Guides.
-- View this [repository's main README](https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/README.md)
-  to see the full list of Cloud APIs that we cover.
-
-[Client Library Documentation]: https://googlecloudplatform.github.io/google-cloud-node/#/docs/redis
-[Product Documentation]: https://cloud.google.com/redis
\ No newline at end of file
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `npm run generate-scaffolding`."
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# [nodejs-redis: Node.js Client](https://github.com/googleapis/nodejs-redis)
+
+[![release level](https://img.shields.io/badge/release%20level-alpha-orange.svg?style&#x3D;flat)](https://cloud.google.com/terms/launch-stages)
+[![CircleCI](https://img.shields.io/circleci/project/github/googleapis/nodejs-redis.svg?style=flat)](https://circleci.com/gh/googleapis/nodejs-redis)
+[![AppVeyor](https://ci.appveyor.com/api/projects/status/github/googleapis/nodejs-redis?branch=master&svg=true)](https://ci.appveyor.com/project/googleapis/nodejs-redis)
+[![codecov](https://img.shields.io/codecov/c/github/googleapis/nodejs-redis/master.svg?style=flat)](https://codecov.io/gh/googleapis/nodejs-redis)
+
+> Node.js idiomatic client for [][product-docs].
```

Comment:
Looks like the generator might have missed a variable.

Reply:
That's because `name` and `short_name` are missing from `.cloud-repo-tools.json`.